packages/interfaces/src/waku.ts

```diff
@@ -25,28 +25,33 @@ export type CreateEncoderParams = CreateDecoderParams & {
   ephemeral?: boolean;
 };
 
+export enum WakuEventType {
```

Comment:
```suggestion
export enum WakuEvent {
```

(unless we already use `WakuEvent` somewhere)

Reply:
I see we're using a similar naming structure for peer manager events below, if it's the standard naming convention for enums in the repo then I guess it's fine but it seems verbose to me